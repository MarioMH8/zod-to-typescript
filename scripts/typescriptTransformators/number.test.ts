import { ZodTypescriptTransformator } from "@scripts/index";
import { z as zod } from "zod";

it("number", () => {
	const result = ZodTypescriptTransformator.convert(zod.number());

	expect(result).toBe("type Zod2ts_1_duplojs = number;");
});
