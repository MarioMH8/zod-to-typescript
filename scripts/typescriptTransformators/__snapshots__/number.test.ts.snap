// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`number 1`] = `"type Zod2ts_0_duplojs = number;"`;
